static/src/js/containers/TimelineContainer/TimelineContainer.scss

```diff
@@ -0,0 +1,7 @@
+.timeline-loading {
```

Comment:
No longer necessary, please remove.

Reply:
Done, removed on the latest commit https://github.com/nasa/earthdata-search/pull/1235/commits/3be40675fd09ea6f1a3dac52d2bb4096b6040599